setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name="OpenFisca-US",
-    version="0.4.0",
+    version="0.4.1",
```

Comment:
Here and in the changelog since this adds functionality - see https://semver.org
```suggestion
    version="0.5.0",
```